llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -21390,6 +21717,23 @@ bool RISCVTargetLowering::splitValueIntoRegisterParts(
       return true;
     }
   }
+
+  if (ValueVT.isRISCVVectorTuple() && PartVT.isRISCVVectorTuple()) {
+    unsigned ValNF = ValueVT.getVectorNumElements();
+    unsigned ValLMUL = std::max(1U, (unsigned)ValueVT.getSizeInBits() /
```

Comment:
Would using `divideCeil` remove the need for maxing with 1?

Reply:
Yes, we can remove the `std::max` and use `divideCail` instead.